src/insert/fill-dynamic-advert-slot.ts

```diff
@@ -29,14 +29,7 @@ const fillDynamicAdSlot = (
 			if (dfpEnv.adverts.has(adSlot.id)) {
 				const errorMessage = `Attempting to add slot with exisiting id ${adSlot.id}`;
 				log('commercial', errorMessage);
-				reportError(
-					Error(errorMessage),
-					{
-						feature: 'commercial',
-						slotId: adSlot.id,
-					},
-					false,
-				);
+				reportError(Error(errorMessage), 'commercial');
```

Comment:
Does DCR raven not support sending additional metadata?

Reply:
Good spot. It looks like [DCR is missing support](https://github.com/guardian/dotcom-rendering/blob/03e25eed03f1af91a99989a002af3e626c333564/dotcom-rendering/src/client/sentryLoader/sentry.ts#L58) for [custom tags](https://docs.sentry.io/platforms/javascript/enriching-events/tags/) and only allows us to pass the `feature`. Let's bring this up with WebX as adding custom tag support to DCR at the capture level would probably be desirable for them too.

Example error with custom tag `slotId`:

https://the-guardian.sentry.io/issues/5666660955/?project=35463&query=is%3Aunresolved+feature%3Acommercial&referrer=issue-stream&statsPeriod=14d&stream_index=0

Reply:
Are we also happy to lose the [`shouldThrow`](https://github.com/guardian/commercial/blob/main/src/utils/report-error.ts#L19) flag?

Reply:
We've now added support for additional metadata in guardian/dotcom-rendering#12280

Reply:
@dskamiotis also here, probs want slotId